delta/rdf/src/test/scala/ch/epfl/bluebrain/nexus/delta/rdf/jsonld/decoder/JsonLdDecoderSpec.scala

```diff
@@ -0,0 +1,98 @@
+package ch.epfl.bluebrain.nexus.delta.rdf.jsonld.decoder
+
+import java.time.Instant
+
+import cats.data.NonEmptyList
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.schema
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.JsonLd
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.{JsonLdContext, RemoteContextResolution}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.decoder.JsonLdDecoderError.DecodingDerivationFailure
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.decoder.JsonLdDecoderSpec.Drink.{Cocktail, Volume}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.decoder.JsonLdDecoderSpec.Menu.DrinkMenu
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.decoder.JsonLdDecoderSpec.{Drink, Menu}
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.decoder.configuration.semiauto._
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.decoder.semiauto._
+import ch.epfl.bluebrain.nexus.delta.rdf.syntax._
+import ch.epfl.bluebrain.nexus.testkit.{CirceLiteral, EitherValuable, IOValues, TestHelpers}
+import org.scalatest.Inspectors
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+import scala.annotation.nowarn
+
+@nowarn("cat=unused")
```

Comment:
remove?

Reply:
No, it complains about non-used Configuration and other decoders, but it is nonsense :)

Reply:
No, it seems necessary.